packages/communication-react/CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ This log was last generated on Tue, 28 Sep 2021 19:19:18 GMT and should not be m
 
 <!-- Start content -->
 
+## [1.0.0-beta.7](https://github.com/azure/communication-ui-library/tree/@internal/react-components_v1.0.0-beta.7)
+
+Mon, 1 Nov 2021 12:57:14 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.0.0-beta.6..1.0.0-beta.7)
+
+### Changes
+
+* Improved how participants are ordered in the `VideoGallery`
+* Improved how video tiles are laid out in the `VideoGallery`
```

Comment:
I would add to this on how they now take into account dominant speakers (Prop `dominantSpeakers` added to VideoGallery to reorder remoteParticipants with new algorithm.), distinguish between video and audio participants, and how the above uses a new horizontal gallery to display participants

Reply:
I am thinking we need to make a larger storybook change here to highlight this information. We can link to storybook later and include the gifys which we would store in our storybook project?